main.js

```diff
@@ -0,0 +1,72 @@
+let todoItems = [];
+
+const addNewToDoItem = () => {
+  let textElement = document.getElementById("todoText");
+
+  if (textElement.value === "") {
+    alert("entering the todo text is mandatory");
+    return;
+  }
+
+  const isItemAlreadyExist = todoItems.some(
+    (item) => item === textElement.value
+  );
+
+  if (isItemAlreadyExist) {
+    alert("the item you've entered is already exist!");
+    return;
+  }
+
+  todoItems.push(textElement.value);
+  localStorage.setItem('items', JSON.stringify(todoItems));
+  render();
+
+  textElement.value = "";
+};
+
+const remove = (itemToRemove) => {
+  let indexToRemove = todoItems.indexOf(itemToRemove);
```

Comment:
since the value is not gonna change, it is better to use `const` here instead of `let`